nautobot/core/templates/template.css

```diff
@@ -0,0 +1,87 @@
+}
+@font-face {
+    font-family: 'Ubuntu';
+    font-style: normal;
+    font-weight: 400;
+    font-display: swap;
+    src: url('{% static 'fonts/Ubuntu-Regular.woff2' %}') format('woff2');
+    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+}
+@font-face {
+    font-family: 'Ubuntu';
+    font-style: normal;
+    font-weight: 500;
+    font-display: swap;
+    src: url('{% static 'fonts/Ubuntu-Medium.woff2' %}') format('woff2');
+    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+}
+@font-face {
+    font-family: 'Ubuntu';
+    font-style: normal;
+    font-weight: 700;
+    font-display: swap;
+    src: url('{% static 'fonts/Ubuntu-Bold.woff2' %}') format('woff2');
+    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+}
+@font-face {
+    font-family: 'Ubuntu Mono';
+    font-style: italic;
+    font-weight: 400;
+    font-display: swap;
+    src: url('{% static 'fonts/UbuntuMono-Italic.woff2' %}') format('woff2');
+    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+}
+@font-face {
+    font-family: 'Ubuntu Mono';
+    font-style: italic;
+    font-weight: 700;
+    font-display: swap;
+    src: url('{% static 'fonts/UbuntuMono-BoldItalic.woff2' %}') format('woff2');
+    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+}
+@font-face {
+    font-family: 'Ubuntu Mono';
+    font-style: normal;
+    font-weight: 400;
+    font-display: swap;
+    src: url('{% static 'fonts/UbuntuMono-Regular.woff2' %}') format('woff2');
+    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+}
+@font-face {
+    font-family: 'Ubuntu Mono';
+    font-style: normal;
+    font-weight: 700;
+    font-display: swap;
+    src: url('{% static 'fonts/UbuntuMono-Bold.woff2' %}') format('woff2');
+    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+}
+h1::before {
+    content: url("{% static 'img/nautobot_chevron_header.svg' %}");
```

Comment:
Let's make this configurable via the `custom_branding_or_static` templatetag - you can refer to https://github.com/nautobot/nautobot/pull/4765/commits/e158eb3035c8cece7c133225a53b8a87ae1889db for an example of the changes needed.

Also, should we:

1. specify an exact height for this image in CSS (allowing the width to vary if overridden with a different image)
2. use the same default `nautobot_chevron.svg` file as is being added in #4765 rather than a separate new file (I know it's a tiny file, but might as well?)

Reply:
1.✅
2. The dimensions for the svg file is not modifiable as a `content:` argument in the `h1::before` tag. But as a `background-image: ` argument it can be. However, that requires us to specify both width and height to align it properly.

Reply:
So width cannot be left unspecified

Reply:
Ah okay, good to know. If we need to retain a separate image file to get the right default dimensions that's OK by me.